src/Tracy/Debugger.php

```diff
@@ -470,6 +470,13 @@ public static function timer($name = NULL)
 		$now = microtime(TRUE);
 		$delta = isset($time[$name]) ? $now - $time[$name] : 0;
 		$time[$name] = $now;
+		if (!self::$productionMode) {
+			static $panel;
+			if (!$panel) {
+				$panel = self::getBar()->getPanel('time');
+			}
+			$panel->data[] = array('name' => $name, 'time' => $now, 'delta' => $delta);
+		}
```

Comment:
Ups, my bad :-) I agree with you now. It was a focus lost.

![](http://media.giphy.com/media/aKYpYXhR6kH1S/giphy.gif)